source/documentation/airflow.md

```diff
@@ -167,12 +171,16 @@ To clone the repository:
 6.  Paste the SSH URL in the **Repository URL** field.
 7.  Select **Create Project**.
 
-### Create a DAG script
+### (9) Create a DAG script
 
 A DAG is defined in a Python script. An two examples of a DAG script are outlined below. One using the `basic_kubernetes_pod_operator` and the other the `KubernetesPodOperator`. Note that the `basic_kubernetes_pod_operator` is a function created and managed by the Data Engineering Team to make it easier to run tasks on your sandboxed airflow instance or the main airflow deployment. If you require something with more functionality the we suggest using the full `KubernetesPodOperator`. Examples of both are below.
 
 #### Example DAG (basic_kubernetes_pod_operator)
 
+<details>
+
+<summary>Click for basic_kubernetes_pod_operator code sample</summary>
```

Comment:
The code within this (and the subsequent block) doesn't render correctly. Presumably because you're nesting markdown within HTML. I would suggest either leaving the code blocks as is or putting them on GitHub and linking to them there.